src/routes/index.js

```diff
@@ -126,7 +135,18 @@ router.post(
           } else {
             user.emailVerified = true;
             user.markModified('emailVerified');
-            user.save();
+
+            const returnedSeaonNumber = await seaonNumber.findOne({}).exec();
+
+            const defaultRankData = new RankData(
+              {
+                username: req.body.username,
+                seasonNumber: returnedSeaonNumber.number,
+              }
+            );
+            await defaultRankData.save();
+            user.currentRanking = defaultRankData._id;
+            await user.save();
```

Comment:
Hmm, did you not have to do something like `RankData.create()`? Or can you just create a new object and call `.save()`?